src/elements/DetailsMenu/DetailsMenu.js

```diff
@@ -152,20 +141,14 @@ const DetailsMenu = ({
             transform: `translateX(${-scrolledWidth}px)`
           }}
         >
-          {detailsMenu.map(
-            tab =>
+          {detailsMenu.map(tab => {
+            const urlArray = match.url.split('/')
+            urlArray[urlArray.length - 1] = tab.id
+            const tabLink = urlArray.join('/')
```

Comment:
I think this is equivalent to:
```suggestion
            const tabLink = urlArray.replace(/^$|([^\/]+$)/, tab.id)
```
It matches either an empty string, or a sequence of one or more characters that are not `/` and are at the end of the string, then replace them with the value of `tab.id`.
```js
'abc/def/ghi'.replace(/^$|([^\/]+$)/, 'jkl')
// => 'abc/def/jkl'
'abc'.replace(/^$|([^\/]+$)/, 'jkl')
// => 'jkl'
''.replace(/^$|([^\/]+$)/, 'jkl')
// => 'jkl'
```

But if you think it's less readable then don't use it.